OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,433 @@
+===========================================
+
+## Contents
+
+1. [Introduction](#1-introduction)
+2. [Prerequisites](#2-prerequisites)
+3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
+4. [Download the required packages/installers](#4-download-the-required-packagesinstallers-patches)
+5. [Required build files](#5-required-build-files)
+6. [Additional build commands](#6-additional-build-commands)
+7. [Steps to Create OUD image](#7-steps-to-create-oud-image)
+8. [Sample Dockerfile generated with imagetool](#8-sample-dockerfile-generated-with-imagetool)
+
+# 1. Introduction
+
+This README describes the steps involved in building an OUD image with the WebLogic Image Tool.
+
+# 2. Prerequisites
+
+The following prerequisites are necessary before building OUD Docker images with Image Tool:
+
+* A working installation of Docker 18.03.1 or later
+* Bash version 4.0 or later, to enable the <tab> command complete feature.
+* JAVA_HOME environment variable set to the location of your JDK e.g:  /scratch/export/oracle/product/jdk
+
+# 3. Setup WebLogic Image Tool
+
+* Download the latest WebLogic Image Tool version from the release [page](https://github.com/oracle/weblogic-image-tool/releases).
+* Unzip the release ZIP file to a desired \<work directory\>.
+
+```
+$ unzip imagetool.zip
+Archive:  imagetool.zip
+   creating: imagetool/
+   creating: imagetool/bin/
+  inflating: imagetool/bin/setup.sh
+  inflating: imagetool/bin/logging.properties
+  inflating: imagetool/bin/imagetool.cmd
+  inflating: imagetool/bin/imagetool.sh
+  inflating: imagetool/LICENSE.txt
+   creating: imagetool/lib/
+  inflating: imagetool/lib/imagetool_completion.sh
+  inflating: imagetool/lib/imagetool.jar
+  inflating: imagetool/VERSION.txt
+  inflating: imagetool/lib/fluent-hc-4.5.12.jar
+  inflating: imagetool/lib/httpclient-4.5.12.jar
+  inflating: imagetool/lib/httpcore-4.4.13.jar
+  inflating: imagetool/lib/commons-logging-1.2.jar
+  inflating: imagetool/lib/commons-codec-1.11.jar
+  inflating: imagetool/lib/httpmime-4.5.12.jar
+  inflating: imagetool/lib/picocli-4.3.2.jar
+  inflating: imagetool/lib/json-20200518.jar
+  inflating: imagetool/lib/compiler-0.9.6.jar
+$
+``` 
+
+* Run the following commands to setup imagetool
+
+```
+$ cd <work directory>/imagetool/bin
+$ source setup.sh
+```
+
+* Execute the following to validate the WebLogic Image Tool:
+
+```
+$ ./imagetool.sh --version
+imagetool:1.9.3
+$
+```
+
+The Image Tool creates a temporary Docker context directory, prefixed by wlsimgbuilder_temp, every time the tool runs. Under normal circumstances, this context directory will be deleted. However, if the process is aborted or the tool is unable to remove the directory, it is safe for you to delete it manually. By default, the Image Tool creates the Docker context directory under the user's home directory. If you prefer to use a different directory for the temporary context, set the environment variable `WLSIMG_BLDDIR`.
```

Comment:
Please wrap `wlsimgbuilder_temp` in back ticks.

Reply:
Updated.